src/azul/__init__.py

```diff
@@ -285,6 +285,8 @@ def qualified_bucket_name(self,
 
     storage_term = 'storage'
 
+    mirror_term = 'datamirror'
```

Comment:
```suggestion
    mirror_term = 'mirror'
```

… unless you think that's going to be problematic.